harvard-lts/etd_alma_monitor_service
# ETD-297: Reinstate JWT Keys in DIMS and add ETD keys

**ETD-297: Reinstate JWT Keys in DIMS and add ETD keys**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-297

# What does this Pull Request do?
calls dims using jwt

# How should this be tested?

this has been deployed to dev

1.  enter dev container
2. launch mongo desktop client, point it to etddev.etds and then search for `{ proquest_id: "0197490463" }` it should bring up one record with an alma_submission_status of **ALMA_DROPBOX**. if it is not that status, reset the status field to that.
3. in the container, type `python3 scripts/invoke-task.py`
4. check the mongo db client again. the alma_submission_status should change to **ALMA**. if you see that then the test was successful.


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
